ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master__upgrade.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-operator-upgrade
+  cron: 30 8 10 * *
+  steps:
+    cluster_profile: aws
+    env:
+      INSTALL_CATSRC: qe-app-registry
+      INSTALL_CHANNEL: stable-v1
+      INSTALL_CSV: cert-manager-operator.v1.11.1
+      UPGRADE_CATSRC: qe-app-registry
+      UPGRADE_CHANNEL: stable-v1
+      UPGRADE_CSV: cert-manager-operator.v1.14.0
```

Comment:
I suggest at least not specify UPGRADE_CSV here. Otherwise even this is automated, you will still need to manually test upgrade for new releases. You need to get it in script.

Reply:
Based on this, you can change job name to e2e-operator-upgrade-to-latest

Reply:
I think it's necessary to keep this option configurable, to provide users with more flexibility.

Reply:
Yes users can. But for your job implemention you should not specify the env var here. Your script should:
Detect if this env var is specified, if yes, use that. If not, use highest.